CHANGELOG.md

```diff
@@ -0,0 +1,161 @@
+## Changelog
+
+### Version 1.19 (2019-05-31)
+
+-   [ JENKINS-42950](https://issues.jenkins-ci.org/browse/JENKINS-42950)
+    - Getting issue details... STATUS  Expanded secret masking to
```

Comment:
The exporter is buggy; you need to manually fix up some Markdown output like this.

Reply:
My fault. Will fix it)

Reply:
Thanks! I do not care too much about the historical changelog since I would expect to switch to Release Drafter anyway.